apps/dapp/src/components/Pages/Core/NewUI/Home.tsx

```diff
@@ -130,18 +124,8 @@ const Home = () => {
   }, [wallet, signer]);
 
   const tradeButtonClickHandler = () => {
-    if (address) {
-      setTradeFormVisible((tradeFormVisible) => !tradeFormVisible);
-    } else {
-      setShowConnect(true);
-    }
-  };
-
-  let targetRef = useRef<HTMLHeadingElement>(null);
-
-  const scrollToContent = () => {
-    // @ts-ignore
-    targetRef.scrollIntoView({ behavior: 'smooth' });
+    if (address) setTradeFormVisible((tradeFormVisible) => !tradeFormVisible);
+    else setShowConnect(true);
```

Comment:
Nit: For readability, I strongly prefer brackets in if/else blocks like this. But just my 2 cents.